share/spice/metar/metar.js

```diff
@@ -0,0 +1,29 @@
+(function (env) {
+    "use strict";
+    env.ddg_spice_metar = function(api_result) {
+        
+        if (!api_result || api_result.Error) {
+            return Spice.failed('metar');
+        }    
+
+        Spice.add({
+            id: "metar",
+            name: 'Weather',
+            data: api_result,
+            meta: {
+                sourceName: "Aviation Weather REST API",
+                sourceUrl: "http://avwx.rest"
+            },
+            normalize: function(item) {
+                return {
+                    title: item["Station"], 
+                    subtitle: item["Time"],
```

Comment:
Great, the API translations (the verbose information) comes in English, so it makes things much easier.

I used an `infoBox` to display the interpreted weather information. Also, I added the complete airport name and city to the subtitle, along with the field elevation (since that does not depend on the weather, I felt more comfortable keeping it out of the `infoBox`).